src/reducers/controls.js

```diff
@@ -87,7 +88,12 @@ export const getDefaultControlsState = () => {
     mapLegendOpen: undefined,
     showOnlyPanels: false,
     showTransmissionLines: true,
-    normalizeFrequencies: true
+    normalizeFrequencies: true,
+    measurementsGroupBy: undefined,
+    measurementsDisplay: "mean",
```

Comment:
I found myself always toggling to raw data, for what it's worth. Perhaps a question for @huddlej?

Reply:
Ah, this was from [feedback](https://bedfordlab.slack.com/archives/C03KWDET9/p1636058549018400?thread_ts=1636057276.018300&cid=C03KWDET9) for the prototype app so it might be very specific to flu titer data.

Reply:
That's a good reason to keep as is than!